[{'error': {'root_cause': [{'type': 'index_not_found_exception', 'reason': 'no such index', 'resource.type': 'index_or_alias', 'resource.id': 'poms', 'index_uuid': '_na_', 'index': 'poms'}], 'type': 'index_not_found_exception', 'reason': 'no such index', 'resource.type': 'index_or_alias', 'resource.id': 'poms', 'index_uuid': '_na_', 'index': 'poms'}, 'status': 404}]